packages/astro/src/integration/index.ts

```diff
@@ -24,9 +24,15 @@ export const sentryAstro = (options: SentryOptions = {}): AstroIntegration => {
         // Will revisit this later.
         const env = process.env;
 
+        const sdkEnabled = {
+          client: typeof options.enabled === 'boolean' ? options.enabled : options.enabled?.client ?? true,
+          server: typeof options.enabled === 'boolean' ? options.enabled : options.enabled?.server ?? true,
+        };
+        const partiallyEnabled = sdkEnabled.client || sdkEnabled.server;
```

Comment:
l: At first glance I assumed this should be XOR, not OR - maybe we can rename this to `sourcemapsNeeded` or something like this?

Reply:
yeah good point, will change it!